frontend/playwright.config.ts

```diff
@@ -218,24 +219,33 @@ function getStorageState(): {
     }>;
   }>;
 } {
-  const storageState = featureFlags;
-
-  if (fs.existsSync(LOGIN_STATE_FILENAME)) {
+  /**
+   * (thuang): Don't login and set `curator` feature flag to `true`,
+   * if `SKIP_LOGIN` is set to `true`.
+   * This way we avoid calling Auth0 and save the rate limit quota
+   */
+  if (!SKIP_LOGIN && fs.existsSync(LOGIN_STATE_FILENAME)) {
```

Comment:
The goal is to minimize the Auth0 test account being used in any minute, so we don't hit the Auth0 rate limiting quota. Since local e2e tests use the same Auth0 test account as GHA, we don't want to set feature flag `curator` to `true` and call Auth0 unnecessarily if `SKIP_LOGIN` is `true`